yarn-project/pxe/src/contract_function_simulator/oracle/private_execution_oracle.ts

```diff
@@ -192,7 +192,17 @@ export class PrivateExecutionOracle extends UtilityExecutionOracle implements IP
    * @returns An app tag to be used in a log.
    */
   public async privateGetNextAppTagAsSender(sender: AztecAddress, recipient: AztecAddress): Promise<Fr> {
```

Comment:
This function will now be doing the orchestration because we will be merging here the info from both ExecutionTaggingIndexCache and the db.